profiles/base/package.use

```diff
@@ -1,11 +1,6 @@
 # Copyright 1999-2017 Gentoo Foundation
```

Comment:
Please move the 'Revert' in the commit message after the prefix, i.e. 'profiles...: Revert "..."'.

Also, the summary of this commit message is too long, breaking stuff attempting to display it. Preferably just ellipsize it before 69.